ui/apps/platform/src/Containers/Integrations/IntegrationTiles/AuthenticationTokensSection.tsx

```diff
@@ -14,6 +14,7 @@ function AuthenticationTokensSection(): ReactElement {
     // Delete unreachable code.
     // Delete request from integration sagas and data from integrations reducer.
     const { currentUser } = useAuthStatus();
+    const { hasReadAccess } = usePermissions();
```

Comment:
hmmm, how did this work before the import was added?

Reply:
It didn't, this is a fix for incorrect rebase